From 19573381ba379cd1dca3c1acb5cfbd68b0578b59 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Mon, 4 Nov 2019 13:16:47 +0700
Subject: [PATCH 095/326] arm64: dts: renesas: r8a774a1: define CPG_AUDIO_CLK_I
 for sound's internal clock source

This patch adds define macro CPG_AUDIO_CLK_I for sound's internal
clock source.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774a1.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
index 341cf71..017a5e8 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774a1.dtsi
@@ -10,6 +10,7 @@
 #include <dt-bindings/clock/r8a774a1-cpg-mssr.h>
 #include <dt-bindings/power/r8a774a1-sysc.h>
 
+#define CPG_AUDIO_CLK_I		R8A774A1_CLK_S0D4
 / {
 	compatible = "renesas,r8a774a1";
 	#address-cells = <2>;
-- 
1.9.1

